components/zigbee/zigbee_attribute.cpp

```diff
@@ -20,6 +20,20 @@ void ZigBeeAttribute::set_attr_() {
   }
 }
 
+void ZigBeeAttribute::set_attr(const std::string &str) {
```

Comment:
Couldn't this be merged with `template<typename T> void ZigBeeAttribute::set_attr(T *value_p)`?

Reply:
Feel free to ignore, see below.